raiden/network/transport/matrix/transport.py

```diff
@@ -1477,14 +1448,51 @@ def partner_joined(fetched_members: Optional[List[User]]) -> bool:
             self.log.debug("Channel room", peer_address=to_checksum_address(address), room=room)
             return room
 
-    def _maybe_create_web_rtc_channel(self, partner_address: Address):
+    def handle_sdp(
```

Comment:
What does `sdp` stand for?

Reply:
[Session Description Protocol](https://tools.ietf.org/html/rfc4566)